packages/react-dev-utils/fileSizeReporter.js

```diff
@@ -0,0 +1,85 @@
+var fs = require('fs');
```

Comment:
Please add a license header and `'use strict'`. You can rebase on master and then `./node_modules/.bin/eslint --ignore-path .gitignore ./` to run linter.